.buildkite/hooks/pre-command

```diff
@@ -15,25 +15,6 @@ git fetch origin master
 # Fetch the git tags to see if that addresses the weird smart build behavior for Habitat
 git fetch --tags --force
 
-# Rebase onto current master to ensure this PR is closer to what happens when it's merged.
-# Only do this if it's actually a branch (i.e. a PR or a manually created build), not a
-# post-merge CI run of master.
-if [[ "$BUILDKITE_BRANCH" != "master" ]]; then
-  git config user.email "you@example.com" # these are needed for the rebase attempt
-  git config user.name "Your Name"
-  master=$(git show-ref -s --abbrev origin/master)
-  pr_head=$(git show-ref -s --abbrev HEAD)
-  github="https://github.com/chef/automate/commit/"
-  if git rebase origin/master >/dev/null; then
```

Comment:
With this on-the-fly rebase, the commits were "fictitious", so could not link to the real ones.